File: java/type3/FindMaxWhile.java
public class FindMaxWhile {
    public static int findMaxV1(int[] values) {
        int maxVal = values[0];
        int i = 1;
        while (i < values.length) {
            if (values[i] > maxVal) maxVal = values[i];
            i++;
        }
        return maxVal;
    }
}

File: javascript/type3/FindMaxWhile.js
function findMaxV1(values) {
    let maxVal = values[0];
    let i = 1;
    while (i < values.length) {
        if (values[i] > maxVal) {
            maxVal = values[i];
        }
        i++;
    }
    return maxVal;
}

module.exports = { findMaxV1 };
